cumulus/parachains/runtimes/glutton/glutton-kusama/src/lib.rs

```diff
@@ -184,6 +193,25 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 
 impl parachain_info::Config for Runtime {}
 
+impl cumulus_pallet_aura_ext::Config for Runtime {}
+
+impl pallet_timestamp::Config for Runtime {
+	type Moment = u64;
+	type OnTimestampSet = Aura;
+	// TODO: set this to 0 so that anyone can produce blocks at anytime?
```

Comment:
I think this should be OK but maybe @Sophia-Gold can confirm. If OK then remove the TODO.

Reply:
What's wrong with `SLOT_DURATION / 2`? I've used this in the upgraded tick and it works fine. Otoh, one would think 0 would be okay as well. I don't totally understand this, but anything less than `SLOT_DURATION` would seem to work.

Reply:
There's nothing wrong with `SLOT_DURATION / 2`, let's leave it as is.